.github/workflows/code-style-linter.yml

```diff
@@ -0,0 +1,55 @@
+name: code-style-linter
+on:
+  push:
+    branches:
+      - master
+      - main
+  pull_request:
+
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
+  # pull-requests: read
+
+jobs:
+  golangci:
```

Comment:
Instead of separated workflow I would rather add this job to run-tests.yaml

Reply:
Done